### DISCUSSION TITLE
Pest v2 memory exhaustion

### DISCUSSION THREAD

**Discussion by sirshaun** [pestphp/pest#726] [General]
Hi! I recently upgraded from laravel 9 to 10 following the laravel upgrade guide as well as the pest upgrade guide, since upgrading I get a "Exit Code: 255(Unknown error)". Wondering if I missed something, any help would be much appreciated.

<img width="817" alt="Screenshot 2023-03-22 at 13 30 42" src="https://user-images.githubusercontent.com/4134605/226892820-45c73126-e431-4258-8e72-e19d3373b67f.png">
<img width="817" alt="Screenshot 2023-03-22 at 13 33 58" src="https://user-images.githubusercontent.com/4134605/226892852-f9f495e0-e1a5-4748-8a1e-854cdcf63f50.png">

**Comment by guillaumeboussion** [comment]
Hey, I have the same problem. Were you able to figure out why ?

**Reply by sirshaun** [reply]
@guillaumeboussion couldn't find any articles or blogs on this matter so i just reverted back to laravel 9 and pest 1

i have since tried it again today and it seems the issue has been resolved, my tests are running just fine. this time around i did the pest v2 upgrade guide first and then the laravel upgrade guide last (maybe that had some effect). hope it ends up working for you as well
